src/post-message.ts

```diff
@@ -202,14 +207,14 @@ const hasWebKitPostMessage = () =>
 const getWebViewPostMessage = (): NovumPostMessage | null => {
     // Android
     if (hasAndroidPostMessage()) {
-        return (jsonMessage) => {
+        return jsonMessage => {
```

Comment:
different prettier versions?
don't we have prettier check in ci?

Reply:
yes, I don't know why vscode take a different version of prettier when I save changes. I had to run prettier with --write to fix this

Reply:
Anyway I think that the husky config doesn't work well else I couldn't commit these changes